client/build_prune.go

```diff
@@ -21,7 +21,16 @@ func (cli *Client) BuildCachePrune(ctx context.Context, opts types.BuildCachePru
 	if opts.All {
 		query.Set("all", "1")
 	}
-	query.Set("keep-storage", strconv.Itoa(int(opts.KeepStorage)))
```

Comment:
We're still keeping `KeepStorage` as deprecated, so we should still set it.

Reply:
Restored this code.